src/main/java/greencity/entities/OpeningHours.java

```diff
@@ -0,0 +1,37 @@
+package greencity.entities;
+
+import greencity.entities.enums.WeekDays;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.persistence.*;
+import java.time.LocalTime;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class OpeningHours {
```

Comment:
Why OpeningHours is a distinct entity?
Maybe it is better to push these fields to the Place entity? (Just thinking, not sure if you need that)
Or do you want to save here opening time for each day of the week for every place? (Then it makes sense)

Reply:
"Or do you want to save here opening time for each day of the week for every place? (Then it makes sense)" - yes. Each day can has different opening time.